EIPS/eip-894.md

```diff
@@ -0,0 +1,36 @@
+## Preamble
+
+    EIP: 894
+    Title: Immutability Enforcement Proposal (IMP)
+    Author: Marius Kjærstad
+    Type: Meta
+    Status: Draft
+    Created: 2018-02-21
+
+
+## Simple Summary
+Provide a standardized response against proposals for bailouts on the offical Ethereum repositories.
+
+
+## Abstract
+Proposals that advocate for any state changes to the Ethereum blockchain should not be merged into the official Ethereum repositories.
+
+
+## Motivation
+The issue of bailouts on the Ethereum blockchain is always controversial. This EIP attempts to raise the barriers for bailouts.
+
+
+## Specification
+If a pull request is opened that advocates for bailouts or implements code that can result in state changes on the Ethereum blockchain, it will be closed.
```

Comment:
This needs some work, will there be some Github-Webhook-Oracle that triggers the proposals to be closed? In its current state, it reads more like you are trying to improve Github, not Ethereum.

Reply:
We only need a standard process for the maintainers to follow.